Simulations/python/run_recipes.py

```diff
@@ -8,42 +8,71 @@
 import runRecipes as rr
 import sys
 import makeCalibPrototypes
-
-
-def runRecipes(argv):
-    
+from pathlib import Path
+
+def runRecipes(inputYAML=Path.joinpath(Path(__file__).parent.parent, 
+                                       "YAML/allRecipes.yaml"), 
```

Comment:
You don't need the `joinpath`, you can do this: `Path(__file__).parent.parent / "YAML" / "allRecipes.yaml"`,nice aye?